apps/admin/src/app-pages/signup-request/ui/SignupRequestPage.tsx

```
@@ -1,6 +1,10 @@
 'use client';
 import { useDebouncedValue } from '@surf/hooks';
 import { TextInput } from '@surf/ui/text-input';
+import { Suspense, useState } from 'react';
+
+import { useDebouncedValue } from '@/shared/hooks/useDebouncedValue';
```

Biome (2.3.13)

[error] Shouldn't redeclare 'useDebouncedValue'. Consider to delete it or rename it.

'useDebouncedValue' is defined here:


(lint/suspicious/noRedeclare)